package.json

```diff
@@ -26,6 +26,7 @@
   "dependencies": {
     "aurelia-dependency-injection": "^1.0.0-beta.1.2.0",
     "aurelia-polyfills": "^1.0.0-beta.1.1.2",
+    "connect-history-api-fallback": "^1.2.0",
```

Comment:
Can you revert this change to the package.json. It's not the cli that needs this dependency, but rather the developer's project. We set that up a different way and I can add it if you can remove this.